app/views/govuk_publishing_components/components/_cookie_banner.html.erb

```diff
@@ -9,7 +9,7 @@
     end
     text = newtext
   else
-    text = "<p class='govuk-body'>#{text}</p>"
+    text = "<p class='govuk-body govuk-body-l'>#{text}</p>"
```

Comment:
Why is `govuk-body-l` applied here and not on line 8?

Reply:
fixed